lib/puppetserver/ca/action/migrate.rb

```diff
@@ -0,0 +1,95 @@
+require 'puppetserver/ca/utils/cli_parsing'
+require 'puppetserver/ca/utils/file_system'
+require 'puppetserver/ca/utils/http_client'
+
+module Puppetserver
+  module Ca
+    module Action
+      class Migrate
+        include Puppetserver::Ca::Utils
+        PUPPETSERVER_CA_DIR = '/etc/puppetlabs/puppetserver/ca'
+
+        SUMMARY = "Migrate the existing CA directory to /etc/puppetlabs/puppetserver/ca"
+        BANNER = <<-BANNER
+Usage:
+  puppetserver ca migrate [--help]
+  puppetserver ca migrate [--config PATH]
+
+Description:
+  Migrate an existing CA directory at /etc/puppetlabs/puppetserver/ca. This is for
```

Comment:
This implies to me that the existing CA directory is at that path. I think this is clearer:
```suggestion
  Migrate an existing CA directory to /etc/puppetlabs/puppetserver/ca. This is for
```